shell/client/grain.html

```diff
@@ -148,9 +148,30 @@
 <template name="grainDebugLogButton">
   <button class="grain-button" title="Show Debug Log" id="openDebugLog">Log</button>
 </template>
+
 <template name="grainBackupButton">
   <button class="grain-button {{#if isLoading}}loading{{/if}}" title="Download Backup" id="backupGrain" disabled="{{#if isLoading}}true{{/if}}">Backup</button>
+  {{#if state.confirming }}
+    {{#modalDialogWithBackdrop onDismiss=cancelBackup}}
+      <form name="confirm">
+        <p>
+         Warning: this grain has access to the following objects.
```

Comment:
I think the copy here is simultaneously a bit technical ("objects") and handwavey ('might not work correctly").  Can we be more specific or less-jargony?

Can we try putting the warning at the beginning, and adding context after the warning?  I worry users will see the technical line and ignore the rest.  Maybe something like:

```html
<p class="warning-message">
  Warning: restoring this backup might not work how you expect it to
</p>
<p>
  When you restore this backup, you'll need to add each of the following to the new grain:
</p>
<ul class="token-list">...</ul>
```

(I'm not thrilled with my proposed text either, especially once we have additional capability types, and knowing that the user making the backup may not have the authority to add the grains to the collection on their own.)